Client/game_sa/CPedSoundSA.h

```diff
@@ -70,15 +70,22 @@ class CPedSoundSA : public CPedSound
     CPedSoundSAInterface* m_pInterface;
 
 public:
+    CPedSoundSA(CPedSoundSAInterface&& pInterface) noexcept
```

Comment:
What is the purpose of rvalue reference here?

Reply:
Nonsence! When you're using a rvalue reference you should you it as a rvalue reference. 

```
CPedSoundSA(CPedSoundSAInterface* pInterface, bool bInstantiate = false) :
        m_pInterface(bInstantiate ? new CPedSoundSAInterface() : pInterface)
{
        if (bInstantiate)
        {
                SetVoiceID(pInterface.m_sVoiceID);
                SetVoiceTypeID(pInterface.m_sVoiceType);
        }
}
```
would be much better